llvm/lib/Target/AMDGPU/AMDGPUISelLowering.cpp

```diff
@@ -3062,6 +3071,26 @@ static bool isCttzOpc(unsigned Opc) {
   return Opc == ISD::CTTZ || Opc == ISD::CTTZ_ZERO_UNDEF;
 }
 
+SDValue AMDGPUTargetLowering::lowerCTLZResults(SDValue Op,
```

Comment:
Nit: I know you've already changed the name once, but isn't this really doing integer result promotion (as in `DAGTypeLegalizer::PromoteIntegerResult`)? So maybe `PromoteIntRes_CTLZ`??

Reply:
I'm happy to change the name if there are no objections? I think "lowerX" was preferred for consistency with the other functions in `ReplaceNodeResults`.